### ISSUE TITLE
Add learning rate finder

### ISSUE THREAD

**Issue by aptrishu** [keras-team/keras#8891]
A learning rate finder can easily be implemented as shown in https://arxiv.org/abs/1506.01186

**Comment by gabrieldemarmiesse** [comment]
I you want to give suggestions of new functions, I think it's more appropriate to go to the repo keras-contrib, suggest it in the issues/mailing list and close the issue here. Make a pull request to keras-contrib if you want to speed up the process. Thank you.
